src/components/Accordion/Accordion.tsx

```diff
@@ -6,11 +6,17 @@ import AccordionItem from './AccordionItem';
 type AccordionProps = {
   children?: JSX.Element | JSX.Element[];
   className?: string;
+  inPageSection?: boolean;
```

Comment:
Added this prop to prevent borders being added to any Accordions used not inside the main page content (e.g. InPageNav).

Previously we had the borders applied through styles on the button inside the Accordion (InPageNav uses the 'chevron' variant, main uses 'plus' variant). This was fine, but because we've moved the borders to the wrapping `cc-accordion` selector* we needed to address this.

*I thought best to move the borders here as I think it's more legible to attach the hack selectors to the top level accordion wrapper, as opposed to a nested button